lib/stacktrace-parser.js

```diff
@@ -50,4 +50,6 @@ var StackTraceParser = {
 };
 
 
-module.exports = StackTraceParser;
+if (typeof module != 'undefined') {
```

Comment:
Hmm, that is enough? The main entry point of the project is the `index.js` which also uses `module.exports` to require that stuff.